app/templates/search/_briefs_pagination_link.html

```diff
@@ -0,0 +1 @@
+<a href="?page={{ page_number }}{% for key, val_list in link_args.lists() %}{% if key != 'page' %}{% for val in val_list %}&{{ key }}={{ val }}{% endfor %}{% endif %}{% endfor %}">{{ link_label }}</a>
```

Comment:
Yes. As this snippet is called in a loop on pagination, the 'page' value in args needs to be overwritten. Tried to remove the 'page' from `request.args.deepcopy()` and add `&page=someNumber` next the the `url_for` but the data structure of the `request.args` is ImmutableMultiDict and did not allow me to remove a key, value from it. I could manually iterate over it and store in a new Dict in the views but did the same thing on template.